tests/code-splitting/subtests/dynamic-import-preloading/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Dynamic import Preloading
+importance: 1
+shortDesc: 'Can code splitting be preloaded?'
+---
+
+# Introduction
+
+While loading the module which has been code spllited, it is important to load its dependencies in parallel to
+optimize the load time.
+
+# The Test
+
+This test checks to see if it's possible to start network request for the dependecies as soon as the code splitted module start loading.
+**index.js**
+
+```js
+setTimeout(async () => {
+  const { shout } =
+    Math.random() < 0.5
+      ? await import('./utilsA.js')
+      : await import('./utilsB.js');
```

Comment:
Seems like this could just be a conditional import of one `utils.js` module with the dice roll - it just needs to not be an unconditional import.

Reply:
@developit I don't get it.

Reply:
Lgtm but I would go with `random() < 0` so that the execution is deterministic. Just `false` doesn't work as this is usually detected by bundlers.